processes/fulfillURLManifest.py

```diff
@@ -80,34 +79,35 @@ def update_manifest(self, metadataObject, bucketName, currKey):
         if metadataJSON != metadataJSONCopy:
             try:
                 fulfillManifest = json.dumps(metadataJSON, ensure_ascii = False)
-                return self.putObjectInBucket(
-                    fulfillManifest, currKey, bucketName
+                return self.s3Client.put_object(Bucket=bucketName, Key=currKey, \
+                                Body=fulfillManifest, ACL= 'public-read', \
+                                ContentType = 'application/json'
```

Comment:
I think we can clean this up a tad: and just put each of the params on a new line and remove each \